service/src/main/java/uk/gov/hmcts/reform/fpl/service/ApplicationDocumentsService.java

```diff
@@ -50,7 +50,8 @@
                 Optional<Element<ApplicationDocument>> documentBefore = findElement(document.getId(),
                     previousDocuments);
 
-                if (documentBefore.isPresent()) {
+                // in the old flow, we allowed other documents with just title and no file
+                if (documentBefore.map(doc -> doc.getValue().hasDocument()).orElse(false)) {
```

Comment:
The old UI, allow Other documents to be added without file attached. In the new collection the file is mandatory. In that case, upon migration, the call would thrown an exception on handleExisitingDocuments()